tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/OrganisationsSteps.cs

```diff
@@ -73,12 +85,48 @@ public async Task ThenTheOrganisationsListIsReturnedWithTheFollowingValues(Table
 
             var organisations = (await _response.ReadBody()).SelectToken("organisations").Select(t => new
             {
-                Name = t.SelectToken("name").ToString(), OdsCode = t.SelectToken("odsCode").ToString()
+                Name = t.SelectToken("name").ToString(),
+                OdsCode = t.SelectToken("odsCode").ToString(),
+                PrimaryRoleId = t.SelectToken("primaryRoleId").ToString(),
+                CatalogueAgreementSigned = t.SelectToken("catalogueAgreementSigned").ToObject<bool>(),
+                Line1 = t.SelectToken("address.line1").ToString(),
+                Line2 = t.SelectToken("address.line2").ToString(),
+                Line3 = t.SelectToken("address.line3").ToString(),
+                Line4 = t.SelectToken("address.line4").ToString(),
+                Town = t.SelectToken("address.town").ToString(),
+                County = t.SelectToken("address.county").ToString(),
+                Postcode = t.SelectToken("address.postcode").ToString(),
+                Country = t.SelectToken("address.country").ToString()
             });
 
             organisations.Should().BeEquivalentTo(expectedOrganisations, options => options.WithStrictOrdering());
         }
 
+        [Then(@"the Organisation is returned with the following values")]
+        public async Task ThenTheOrganisationIsReturnedWithTheFollowingValues(Table table)
+        {
+            var expectedOrganisation = table.CreateSet<OrganisationTable>().FirstOrDefault();
+
+            var org = await _response.ReadBody();
```

Comment:
Try to make variable names appropriate to the object you are initialising to e.g. responseBody. So when reading the code one doesn't have to context switch & will hopefully follow it better.